### PR TITLE
Make development docker Work Again

### PR DESCRIPTION
**Pull request by vikoivun**
Cleanups and upgrades for dev Dockerization

Postgres was upgraded to version 10, with Postgis 2.5.
Added a nifty endpoint with migrate and runserver commands.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -24,17 +16,16 @@ services:
...
             - "8000:8000"
         depends_on:
             - postgres
-            - elastic
         container_name: linkedevents-backend
 
 volumes:
-        postgres-data-volume:
-        django-media-volume:
+        linkedevents-postgres:
```
Comment (igordavydsson) [inline docker-compose.yml]: docker prefixes all volumes with the folder's name, so the volume will be called `linkedevents_linkedevents-postgres`

File: docker/django/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 FROM python:3.6
 
-ENV PYTHONUNBUFFERED 0
+# Force flush all messages immediately, not really Docker specific
+ENV PYTHONUNBUFFERED 1
```
Comment (igordavydsson) [inline docker/django/Dockerfile]: why does no one like unbuffered output? 😓 userful for development

File: docker-compose.yml (hunk 1 above)
Comment (vikoivun) [reply docker-compose.yml]: Argh. Forgot about that. I'll kick out the prefix. Is the `volume` postfix useful in some situations? Ie. are they listed in context where it is not immediately obvious they are volumes?

Comment (igordavydsson) [reply docker-compose.yml]: I think `volume` postfix can be dropped.

Comment (sam-hosseini) [reply docker-compose.yml]: And Ville, if you could please pull latest changes before working again on commits, it'd be great as my Git global config was making my commits anonymous (which was intentional 😅) but I fixed it and had to do a force push (only and only this time) 👍

Comment (vikoivun) [reply docker-compose.yml]: All right. It seems to have confused git a wee bit, but nothing major.

**Comment by sam-hosseini** [comment]
https://github.com/City-of-Helsinki/linkedevents/pull/356 has been inspired by this PR and therefore includes all the changes already, hence the conflicting files. If you see fit, this can be closed :)

**Pull request closed without merging**